src/main/java/jenkins/plugins/logstash/persistence/BuildData.java

```diff
@@ -65,7 +64,7 @@
  */
 public class BuildData {
   // ISO 8601 date format
-  public transient static final DateFormat DATE_FORMATTER = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ");
+  public transient static final FastDateFormat DATE_FORMATTER = FastDateFormat.getInstance("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
```

Comment:
While we'r at it can you restrict it's visibility? The formatter has been made private in #40

Reply:
I would like to keep changes in this PR to the findbugs issues. providing a method to access the formatter should be done in a separate PR.